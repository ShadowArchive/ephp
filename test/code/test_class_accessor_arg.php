<?php

$mode = 10;
if ($mode != 0) $block = preg_replace("/\r\n|\r|\n/", $this->_eol_code[$this->OS_local], $block);

