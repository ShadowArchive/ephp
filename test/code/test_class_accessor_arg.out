
Fatal error: Using $this when not in object context in {{CWD}}/test/code/test_class_accessor_arg.php on line 4
